emails/management/commands/process_emails_from_sqs.py

```diff
@@ -117,50 +128,72 @@ def init_vars(
         )
         assert 0 <= self.verbosity <= 3
 
+    def help_message(self, text, setting_name):
```

Comment:
suggestion (non-blocking): can this be `help_message_for_setting` to make it more obvious what the 2nd argument is when calling this? E.g., `self.help_message_for_setting("Path to healthcheck JSON file.", "PROCESS_EMAIL_BATCH_SIZE")`